atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/Schemas.java

```diff
@@ -105,30 +104,23 @@ public static boolean isTableNameValid(String tableName) {
      * (e.g., it is not the responsibility of this method to perform schema
      * upgrades).
      */
-    public static void createTablesAndIndexes(Schema schema, KeyValueService kvs) {
+    public static void createTablesAndIndexes(Schema schema, DdlManager ddlManager) {
         schema.validate();
-        createTables(kvs, schema.getTableDefinitions());
-        createIndices(kvs, schema.getIndexDefinitions());
+        createTables(ddlManager, schema.getTableDefinitions());
+        createIndices(ddlManager, schema.getIndexDefinitions());
     }
 
-    public static void deleteTablesAndIndexes(Schema schema, KeyValueService kvs) {
-        schema.validate();
-        kvs.dropTables(getExistingTablesAlsoPresentInSchema(schema, kvs));
-    }
+    // To be removed
 
-    /** intended for use by tests. **/
-    public static void truncateTablesAndIndexes(Schema schema, KeyValueService kvs) {
-        schema.validate();
-        kvs.truncateTables(getExistingTablesAlsoPresentInSchema(schema, kvs));
+    public static void createTableLegacy(Schema schema, KeyValueService kvs, TableReference tableRef) {
```

Comment:
My thinking here was to centralize this bit of logic, until we figure out a better way for things like Sweep to create their underlying tables (which should ONLY be created using KeyValueService NOT DdlManager).